internal/remediation/override.go

```diff
@@ -279,6 +279,58 @@ func getVersionsGreater(ctx context.Context, cl client.DependencyClient, vk reso
 	return versions[offset:], nil
 }
 
+// comparisonFunctionWithWorkarounds returns a version comparison function with special behaviour for specific packages,
+// producing more desirable ordering using non-standard comparison.
+func comparisonFunctionWithWorkarounds(vk resolve.VersionKey) func(resolve.Version, resolve.Version) int {
```

Comment:
can we make this re-usable (for example `osv-scanner update` may need this as well)?

Reply:
Added as a TODO, it'll probably need to be moved to avoid circular import problems.